src/icons/iconAccount.jsx

```diff
@@ -109,4 +109,6 @@ export default function IconAccount({ type, ...props }) {
       </svg>
     );
   }
+
+  return <></>;
```

Comment:
Do we wanna actually render an empty element? I think we can just return null in this case.